bullet_train/lib/bullet_train/resolver.rb

```diff
@@ -251,10 +251,71 @@ def locale_path
       nil
     end
 
+    # In this search, we prioritize files in local themes,
+    # and then look in theme gems if nothing is found.
+    def js_or_stylesheet_path
+      file_name = @needle.split("/").last
+
+      # Prioritize the current theme, and fall back to
+      # the default `light` theme if nothing is found locally.
+      puts "Searching under current theme: #{current_theme.blue}"
+
+      # If the file name doesn't include the theme name (i.e. - light.tailwind.css),
+      # we append the theme name to the path to match something like "app/assets/javascript/foo/".
+      append_theme_name = !file_name.match?("#{current_theme}.")
+      asset_globs = ["js", "css"].map do |extention|
+        path = "app/assets/"
+        if append_theme_name
+          path += case extention
+          when "js"
+            "javascript/"
+          when "css"
+            "stylesheets/"
+          end
+          path += "#{current_theme}/"
+        end
+        path + "**/*.#{extention}"
+      end
+
+      # Ensure we include JavaScript files that exist under the app's root directory.
+      asset_globs << "*.js"
+
+      # Search for the file.
+      files = Dir.glob(asset_globs).reject { |file| file.match?("/builds/") }
+      absolute_file_path = files.find { |file| file.match?(/#{file_name}$/) }
+
+      if absolute_file_path.nil? # then search for it in its respective gem.
+        gem_path = `bundle show bullet_train-themes-#{current_theme}`.chomp
+
+        # Fall back to `light` theme if no gem is available.
+        # (The developer might just be trying to find stylesheets that exist in `light`.)
+        if gem_path.empty?
+          gem_path = `bundle show bullet_train-themes-light`.chomp
+          return nil if gem_path.empty?
+        end
```

Comment:
I wonder if we should condense and unify these branches? I don't think the two \`\` calls would be that expensive if the latter needn't have run.

```suggestion
        # Check the current them but fall back to `light` theme if no gem is available.
        # (The developer might just be trying to find stylesheets that exist in `light`.)
        gem_path = [`bundle show bullet_train-themes-#{current_theme}`, `bundle show bullet_train-themes-light`].map(&:chomp).find(&:present?)
        return nil unless gem_path
```